lib/src/migrator.dart

```diff
@@ -126,10 +127,45 @@ class _Migrator extends RecursiveStatementVisitor implements ExpressionVisitor {
   /// Adds a namespace to any function call that require it.
   void visitFunctionExpression(FunctionExpression node) {
     visitInterpolation(node.name);
+    _patchNamespaceForFunction(node.name.asPlain, (name, namespace) {
+      _currentMigration.patches.add(Patch(node.name.span, "$namespace.$name"));
+    });
     visitArgumentInvocation(node.arguments);
 
-    if (node.name.asPlain == null) return;
-    var name = node.name.asPlain;
+    if (node.name.asPlain == "get-function") {
+      var nameArgument =
+          node.arguments.named['name'] ?? node.arguments.positional.first;
+      if (nameArgument is! StringExpression ||
+          (nameArgument as StringExpression).text.asPlain == null) {
+        print(nameArgument.span.message(
+            "WARNING - get-function call may require \$module parameter"));
+        return;
+      }
+      var fnName = nameArgument as StringExpression;
+      _patchNamespaceForFunction(fnName.text.asPlain, (name, namespace) {
+        var span = fnName.span;
+        if (fnName.hasQuotes) {
+          span = span.file.span(span.start.offset + 1, span.end.offset - 1);
+        }
+        _currentMigration.patches.add(Patch(span, name));
+        var beforeParen = node.span.end.offset - 1;
+        _currentMigration.patches.add(Patch(
+            node.span.file.span(beforeParen, beforeParen),
+            ", \$module: $namespace"));
```

Comment:
Nit: Emit quotes around the namespace, in case it's a color name or something. There is actually [a Sass library named `true`](https://github.com/oddbird/true), so this isn't even purely hypothetical.